mslib/msui/mss_pyui.py

```diff
@@ -416,6 +418,15 @@ def closeEvent(self, event):
                 self.mscolab_window.close()
             if self.config_editor is not None:
                 self.config_editor.close()
+            """
```

Comment:
I think with self.tv_window it is clear what it is, also later the history will connect to the PR. Please compress this to a single comment line